jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/RuntimeDataServiceImplTest.java

```diff
@@ -1239,7 +1235,7 @@ public void testGetTasksAssignedAsPotentialOwnerSortedByAlias() {
     @Test
     public void testTasksByStatusByProcessInstanceIdPagingAndFiltering() {
 
-    	Long pid = processService.startProcess(deploymentUnit.getIdentifier(), "org.jbpm.writedocument");
+    	processInstanceId = processService.startProcess(deploymentUnit.getIdentifier(), "org.jbpm.writedocument");
```

Comment:
I would rather use a variable which belongs to the test itself as it's less error-prone to tests influencing each other.

Reply:
I agree with the principle, but in practise global variable approach is used for all other test and allow defining commom method in parent class (changing that at this stage is not providig any benefit). This line was modified  to align this test with the other ones.

Reply:
OK, makes sense.